docs/best-practices.md

```diff
@@ -132,3 +132,16 @@ the package if they want to experience as smooth upgrade as possible.
 If you're writing an actor for the RHEL 7 to RHEL 8 in-place upgrade workflow and you really need to have some package
 installed, then the only acceptable packages to depend on are the the ones available in the minimal installation of
 RHEL 7 (packages from the @core group + their dependencies).
+
+## When something goes wrong stop the execution of your actor in the right way
+
+If your actor get unexpected input or result of some operation, you must notify the framework about it. For that use these exceptions with a descriptive message:
+- [StopActorExecution](https://leapp.readthedocs.io/en/latest/pydoc/leapp.html#leapp.exceptions.StopActorExecution) - will stop the current actor execution
+- [StopActorExecutionError](https://leapp.readthedocs.io/en/latest/pydoc/leapp.html#leapp.exceptions.StopActorExecutionError) - will stop the current actor execution, but also notify the framework that an error has occurred and can influence the result of the upgrade process.
```

Comment:
Do not mention "upgrade" here (and down below). The best practices should be not only for the upgrade process, but for any other (future) workflow as well.
So, change "upgrade process" to "workflow execution", or something like that.